instrumentation/opentelemetry-instrumentation-elasticsearch/test-requirements-8.txt

```diff
@@ -0,0 +1,23 @@
+asgiref==3.7.2
+attrs==23.2.0
+Deprecated==1.2.14
+elasticsearch==8.12.1
+elasticsearch-dsl==8.12.0
+elastic-transport==8.12.0
```

Comment:
Is the idea that for 8 we support a recent release (8.12)) but not the latest (8.13)?

Reply:
That is for handling the native instrumentation in following PRs, more tests were failing because of double spans :)